### DISCUSSION TITLE
[BUG] why my all plot go to "Not found Plots" and cant farm

### DISCUSSION THREAD

**Discussion by controlnow** [Chia-Network/chia-blockchain#3184] [General]
when I reboot my computer
I found all my own plots go to "not found plots"
I cant use them to farm
I tried to reinstall the software, reboot the computer again, and recreate new wallet
all this are fail, I cant add my plot again
all of them go to Not found Plots
who can help me?
![image](https://user-images.githubusercontent.com/82243000/114247338-7119f400-99c7-11eb-9fde-f414fb5e0e7c.png)

**Comment by THe-SP00K** [comment]
Support questions are best handled in the Keybase forum - there are always community members online willing to help troubleshoot https://keybase.io/team/chia_network.public

**Comment by nonacosa** [comment]
I have the same  questions, I use NAS network mapping to windows or macOS

![image](https://user-images.githubusercontent.com/14212375/115135824-0f542c80-a04e-11eb-987f-ec124a1cc0a7.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
